src/main/scala/cromwell/binding/WdlBinding.scala

```diff
@@ -115,6 +114,11 @@ object WdlBinding {
     }
   }
 
+  /* All MemberAccess ASTs that are not contained in other MemberAccess ASTs */
+  def findTopLevelMemberAccesses(expr: AstNode) = WdlBinding.findAstsWithTrail(expr, "MemberAccess").filter {
+    case (k, v) => !(v map {case a:Ast => a.getName} contains "MemberAccess")
```

Comment:
I don't recall ever seeing the "bunch of partial functions inside combinator closures" issue addressed in a style guide, although given that you simply don't see it very often at all it could be that no one ever felt the need to say it explicitly :)